pyvelox/pyvelox.h

```diff
@@ -396,6 +517,32 @@ static void addVectorBindings(
       .def("append", [](VectorPtr& u, VectorPtr& v) { appendVectors(u, v); })
       .def("resize", &BaseVector::resize);
 
+  py::class_<ArrayVector, ArrayVectorPtr, BaseVector>(
+      m, "ArrayVector", py::module_local(asModuleLocalDefinitions))
```

Comment:
We should also have constructors that allow us to create any complex vector with a base vector , with offsets/lengths etc provided.